### DISCUSSION TITLE
Dump WS Initializing… Stop

### DISCUSSION THREAD

**Discussion by PokeyMinch** [sanni/cartreader#170] [Need help]
![image](https://user-images.githubusercontent.com/85623113/121484409-2722ae00-ca0a-11eb-92c7-0dd8f2d3006f.jpeg)
 A message ”Initializing...“ is displayed. 
It has been an hour and I still don't have a change.
Dose anyone dump Wonder Swan cartredge is available in  cartreader since version 5.5?
![image](https://user-images.githubusercontent.com/85623113/121488968-8d113480-ca0e-11eb-90f6-7f83df5bb30c.jpeg)
Thanks

**Comment by sanni** [comment]
Sorry I don't have that adapter nor any games for WS. 
What was the last firmware version it worked with? Maybe I can check the code to see what has changed since then.

**Reply by PsyK0p4T** [reply]
@PokeyMinch any news about your tests since ?
Double check your solders and tell us what version you are using, and if it worked for you earlier on another version.
I'm grabbing the needed stuff to build mine at the moment, I'll maybe be able to help in a few weeks.

**Comment by PokeyMinch** [comment]
![image](https://user-images.githubusercontent.com/85623113/126092591-fea62ed3-cf01-4677-b7a2-03b4a8851c7b.jpeg)
![image](https://user-images.githubusercontent.com/85623113/126092601-de5d0bba-29de-4df8-952f-93ad9aef71bf.jpeg)
Use tester to check the solders,no bridge ,but not work normally.
So, I retried soldering put thermal pressure.
And I updated cartreader to ver6.4.
As a result, the problem has been resolved.

Apologies for the trouble.

I use HITACHI 16V/10uF Tantalum capacitors(TMCTXC1C106MTRF).
![image](https://user-images.githubusercontent.com/85623113/126092647-35a053d2-dc0c-4676-b473-9a2c6aabd4af.jpeg)

**Reply by PsyK0p4T** [reply]
Digging up this thread a bit. I finally built my WS adapter, and faced to this init error too ^^
So for any future people getting this, few tips : 

- Don't wait long in front of your screen if the init message is displayed. The program will just try to unlock the MMC over and over again (do while loop), so if nothing new is displayed after just a few seconds, don't waste your time and turn your unit off. It means you got a hardware problem somewhere.
- Double check your solderings. Really. Their are really small and errors can come fast. With a tester, make sure that WS slot pins are all independant and not touching to a neighbour pin. Then check that every WS slot pins are linked to one of SNES slot pin.
- And finally, use correct capacitors. Here are the ones I use for mine : https://www.ebay.com/itm/362718382714 (6.3V 10uF case P)

Happy building !

**Reply by rodrigosieghart** [reply]
@PokeyMinch @PsyK0p4T hi this one can dump a Ws and WSC cartridge save file to pc ? You know something about a pocket version for them ?

**Reply by PsyK0p4T** [reply]
I never tried to play with WS game saves for now. But according to the source code, SRAM and EEPROM saves should be handled. Dumping saves to your sd, and restore them to the cart aswell.
